var/spack/repos/builtin/packages/silo/package.py

```diff
@@ -68,8 +68,10 @@ class Silo(AutotoolsPackage):
     # autoconf-archive in 2011
     patch('configure-AX_CHECK_COMPILE_FLAG.patch')
 
-    # API changes in 1.13 cause breakage
-    conflicts('hdf5@1.13:', when='+hdf5')
+    # API changes in hdf5-1.13 cause breakage
+    # See https://github.com/LLNL/Silo/pull/260
+    patch('hdf5-113.patch', when='@4.11:')
```

Comment:
Make this conditional on the newer hdf5? Better not to patch code if we can help it

Reply:
(To clarify: if the patch is unconditional, it'll invalidate every current installation of silo.

```suggestion
    patch('hdf5-113.patch', when='@4.11: ^hdf5@1.13:')
```